src/ESUGApp-Component/ERItemNav.class.st

```diff
@@ -73,5 +103,15 @@ ERItemNav >> renderContentOn: html [
 			html mdlIcon
 				class: 'mdl-color-text--white';
 				with: [ html text: icon ].
-				html text:  label . ]
+				html text:  label . 
+				self setBadge:html.
+				]
+]
+
+{ #category : #accessing }
+ERItemNav >> setBadge:html [
+	|notifs|
+	notifs:= self notifCount.
+	(notifs > 0 ) ifTrue: [ html mdlBadge dataBadge: notifs; title: notifs asString,' unaccepted talks.'. ]
```

Comment:
```suggestion
	(self notifCount > 0 ) ifTrue: [ html mdlBadge dataBadge: notifs; title: notifs asString,' unaccepted talks.'. ]
```

Reply:
Since the temporary is used only once and has an explicit name, we can remove the temporary